services/worker-manager/src/api.js

```diff
@@ -103,7 +102,14 @@ builder.declare({
     'worker-manager:provider:<providerId>',
   ]},
   description: [
-    'Given an existing worker pool definition, this will modify it and return the new definition.',
+    'Given an existing worker pool definition, this will modify it and return',
+    'the new definition.',
+    '',
+    'To delete a worker pool, set its `providerId` to `"null"` and empty its',
```

Comment:
To avoid confusion, what if we called it `"null-provider"`

Reply:
Or `"expire-provider"` or... something like that?

Reply:
I like null-provider..